docs/projects/skeleton.rst

```diff
@@ -9,14 +9,18 @@ Implementation Plan
 The Main functions of the tool are:
 
 - Create and Transform Bones:
-    - Single click would add a default bone with all the handles. 
+    - Double-click would add a default bone with all the handles. 
```

Comment:
I think it's okay to leave this with "single-click". You can put a clarification: "single-click on an empty space of workarea". ^__^